app/controller/grid/Grid.js

```diff
@@ -688,5 +686,120 @@ Ext.define('CpsiMapview.controller.grid.Grid', {
         };
 
         viewModel.setStores(stores);
+    },
+
+    /**
+     * Applies preset filters from the configuration
+     * to the grid.
+     */
+    onApplyPresetFilters: function () {
```

Comment:
Call this automatically when the grid is created?

Reply:
yes, we can do that. Shall the preset filter always be applied directly? Or shall we have a config for that?